README.md

```
@@ -89,5 +89,41 @@ Follow these instructions to get a copy of the project running on your local mac
 
 ```bash
-git clone [https://github.com/AshutoshRaj1260/FoodGram.git](https://github.com/AshutoshRaj1260/FoodGram.git)
+git clone https://github.com/AshutoshRaj1260/FoodGram.git
 cd FoodGram
+```
+
+## 🐳 Running with Docker (Recommended)
+
+The easiest way to get FoodGram up and running is using Docker Compose. This will set up both the backend and frontend in containers.
+
+### 1. Prerequisites
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)